NEW_README.md

```diff
@@ -16,9 +18,15 @@ This will configure multiple nodes.  In the simplest case you need 2 nodes: a Pu
 * Execute:
     * export BEAKER_HOSTS=\<your hosts file>
```

Comment:
You don't need to do this anymore...  at least it is optional.   It should either be mentioned as optional, or moved up to the section about doing something custom.